src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -259,7 +265,9 @@ const ChatItemMyTextV2 = ({
                     onPress={() => Linking.openURL(item.file_path)}>
                     <View style={styles.attachmentFileContainer}>
                       <View style={styles.attachmentFileContent}>
-                        <Text style={styles.attachmentFileName}>{item.file_name}</Text>
+                        <Text style={styles.attachmentFileName} numberOfLines={1}>
+                          {item.file_name}
+                        </Text>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding `numberOfLines={1}` ensures that the filename will not wrap and take up more than one line. This is a good change for maintaining consistent layout and preventing potential overflow issues.


<!-- This is an auto-generated comment by OSS CodeRabbit -->